packages/components/accordion/Accordion.mdx

```diff
@@ -44,35 +48,49 @@ The accordion has two variations that define the alignment of the chevron icon:
 </Accordion>
 ```
 
+### Using it with other components
+
 Other typographic components can be passed as the accordion's title and anything can be used as the accordion's content. For example:
 
 ```jsx
 <Accordion>
   <Accordion.Item
-    title={<SectionHeading>What payment methods do you accept?</SectionHeading>}
+    title={
+      <SectionHeading marginBottom="none">
+        What payment methods do you accept?
+      </SectionHeading>
+    }
   >
-    <Text marginBottom="spacingS">
-      Customers on the Team tier can pay with a credit card (American Express,
-      MasterCard or Visa). Enterprise customers have the choice of paying with a
-      credit card or wire transfer.
-    </Text>
-    <Button onClick={() => console.log('clicked!')}>Accordion’s button</Button>
+    <Stack flexDirection="column">
+      <Text marginBottom="spacingS">
+        Customers on the Team tier can pay with a credit card (American Express,
+        MasterCard or Visa). Enterprise customers have the choice of paying with
+        a credit card or wire transfer.
+      </Text>
+      <Button onClick={() => console.log('clicked!')}>
+        Accordion’s button
+      </Button>
+    </Stack>
   </Accordion.Item>
 </Accordion>
 ```
 
-## Content recommendations
-
-- The title should be a short message that summarize the content of the accordion.
-- Anything can be passed as the content of the accordion, but often organizing it with Texts and Tables would be enough.
-- When using headings, be mindful about the heading levels. The header is a Subheading with `h2` tag.
-
-## Props
+## Props (API reference)
 
 ### Accordion
 
 <Props of="Accordion" />
 
-### AccordionItem
+### Accordion.Item
 
 <Props of="AccordionItem" />
```

Comment:
Props from `Accordion.Item` are not rendered for some reason on the preview

Reply:
I think you need to clean your cache

![Screenshot 2021-11-26 at 13 35 18](https://user-images.githubusercontent.com/6597467/143581993-46522a7b-1f2f-4cf2-9e96-08d3e5a22330.png)